lib/jobs/dell-racadm-tool-job.js

```diff
@@ -0,0 +1,126 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var di = require('di');
+
+module.exports = racadmToolJobFactory;
+di.annotate(racadmToolJobFactory, new di.Provide('Job.Dell.RacadmTool'));
+di.annotate(racadmToolJobFactory, new di.Inject(
+    'Job.Base',
+    'JobUtils.RacadmTool',
+    'Logger',
+    'Util',
+    'Assert',
+    'Promise',
+    '_',
+    'Services.Encryption',
+    'Services.Lookup',
+    'Constants',
+    'Services.Waterline'
+));
+
+function racadmToolJobFactory(
+    BaseJob,
+    racadmTool,
+    Logger,
+    util,
+    assert,
+    Promise,
+    _,
+    encryption,
+    lookup,
+    Constants,
+    waterline
+) {
+    var logger = Logger.initialize(racadmToolJobFactory);
+    /**
+     *
+     * @param {Object} options
+     * @param {Object} context
+     * @param {String} taskId
+     * @constructor
+     */
+    function racadmToolJob(options, context, taskId) {
+        racadmToolJob.super_.call(this, logger, options, context, taskId);
+
+        this.nodeId = this.context.target;
+        this.action = options.action;
+        this.cifsInfo = {user: options.server_username, password: options.server_password ,
+            filePath: options.server_filePath};
```

Comment:
Hound CI reports Errors about naming on server_xxx, change to example like serverUsername/serverPassword/serverFilepath.
One question, what happens if no username/password are delivered? They are optional parameters